### DISCUSSION TITLE
Controlling one entity with two elements

### DISCUSSION THREAD

**Discussion by funzie19** [ExperienceLovelace/ha-floorplan#159] [General]
I have one switch which turns on a fan extractor on two bathrooms. In Floorplan I have setup an icon on each bathroom and I want to be able to turn on the fan by clicking either icon. I would also like the SVG image to spin.

I've been able to do this with one entity but not two. Inkscape does not allow me to re-use the ID of an element. Is there anyway around this?

Thanks.

**Comment by OzGav** [comment]
Yes tricky. You can select both icons in Inkscape and GROUP them which will allow you to touch either one but I don’t think the spin will work correctly.  So my best idea at the moment is for you to setup a `binary_sensor` for one and use automations to keep both in sync with each other.

**Comment by funzie19** [comment]
Thanks for the reply. I'll give that a try.
